tests/Test.ADAL.NET.Unit/NonInteractiveTests.cs

```diff
@@ -128,6 +129,28 @@ public class NonInteractiveTests : AdalTestsBase
         [TestMethod]
         [Description("WS-Trust Address Extraction Test")]
         [TestCategory("AdalDotNet")]
+        public async Task WsTrustPolicyExtraction()
+        {
+            XDocument mexDocument = null;
+            using (Stream stream = new FileStream("TestMex2005.xml", FileMode.Open))
+            {
+                mexDocument = XDocument.Load(stream);
+            }
+
+            Verify.IsNotNull(mexDocument);
+            Dictionary<string, MexPolicy> policies = MexParser.ReadPolicies(mexDocument);
+            Verify.IsNotNull(policies);
+            Verify.IsTrue(policies.Count == 2);
```

Comment:
Are you verifying specifically (and separately) for WIA policies and usernamepassword policies? I don't see that, and it seems like you should test for each specifically.